tracegen/internal/tracegen/config.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
since this is not the code from Jaeger verbatim, a better copyright reference is
```
// Copyright (c) 2020 The OpenTelemetry Authors.
// Copyright (c) 2018 The Jaeger Authors.
```